launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -156,7 +158,9 @@ def _load_in_sequence(
         """
         next_composable_node_description = composable_node_descriptions[0]
         composable_node_descriptions = composable_node_descriptions[1:]
-        self._load_node(next_composable_node_description, context)
+        thread = threading.Thread(
+             target=self._load_node, args=(next_composable_node_description, context))
+        thread.start()
```

Comment:
@jacobperron and I talked in person about this, but just to summarize, I think this thread needs to be joined at some point and we need to indicate to the launch service that this action has some asynchronous behavior, by overriding the `get_asyncio_future()` method: https://github.com/ros2/launch/blob/4333910ba41977f3ab9867082c86a8698ada3a16/launch/launch/launch_description_entity.py#L137

Reply:
On second look, I realized that the code was already making asynchronous calls and providing futures to the launch service with [add_completion_future](https://github.com/ros2/launch_ros/blob/04ce971240eb39f43ca46cf3cc0b3cef4501f4a1/launch_ros/launch_ros/actions/load_composable_nodes.py#L161-L164). The bug was that the first call was not following the same pattern, which I've fixed in 04ce971

PTAL